waiter/src/waiter/request_log.clj

```diff
@@ -60,8 +57,12 @@
 
 (defn log-request!
   "Log a request"
-  [request response]
-  (log (merge (request->context request) (response->context response))))
+  [request response additional-context]
+  (log (merge (request->context request)
+              (when (map? response)
```

Comment:
When is it not a map?  I was expecting the map to come back with all sorts of goodies.

Reply:
Websocket handlers are not required to return a map. E.g. the inter-router metrics connections do not return a map. Another option is to create a separate request-log wrapper that only looks into the request map and use it for websocket handlers.

Reply:
We should update the metrics handler to return a map so we can log which router authenticated and any other things we'd might like to log.  Would you rather do that in a separate PR?

Reply:
Separate PR, we can merge that before merging this one.